pkg/app/bundle_event_handler.go

```diff
@@ -27,12 +29,13 @@ func (h *bundleEventHandler) OnDelete(obj interface{}) {
 
 func (h *bundleEventHandler) handle(obj interface{}) {
 	in := obj.(*smith.Bundle)
-	out := &smith.Bundle{}
 
-	if err := smith.DeepCopy_Bundle(in, out); err != nil {
+	out, err := h.scheme.DeepCopy(in)
+	if err != nil {
 		log.Printf("[TEH] Failed to do deep copy of %#v: %v", in, err)
 		return
 	}
-	log.Printf("[TEH] Rebuilding %s/%s bundle because it was added/updated", out.Metadata.Namespace, out.Metadata.Name)
-	h.processor.Rebuild(out)
+
+	log.Printf("[TEH] Rebuilding %s/%s template because it was added/updated", out.(*smith.Bundle).Metadata.Namespace, out.(*smith.Bundle).Metadata.Name)
+	h.processor.Rebuild(out.(*smith.Bundle))
```

Comment:
Could extract `out.(*smith.Bundle)` into a variable and use it in all three places instead for better readability.